ext/opcache/ZendAccelerator.c

```diff
@@ -3270,7 +3270,11 @@ static zend_result accel_post_startup(void)
 			 || zend_jit_startup(ZSMMG(reserved), jit_size, reattached) != SUCCESS) {
 				JIT_G(enabled) = 0;
 				JIT_G(on) = 0;
-				zend_accel_error(ACCEL_LOG_WARNING, "Could not enable JIT!");
+				/* The JIT is implicitly disabled with opcache.jit_buffer_size=0, so we don't want to
+				 * emit a warning here. */
+				if (JIT_G(buffer_size) != 0) {
```

Comment:
To be entirely honest, I'm not a huge fan of how JIT is disabled by default using the `opcache.jit_buffer_size=0` default, instead of `opcache.jit=0`.

Do you think it would be to change the defaults of `opcache.jit` to `0`, and `opcache.jit_buffer_size` to a sane default like `16m` at least on master (I'd really love to make the same change on PHP-8.1 though...)?

Reply:
I agree that the INI values are sub-optimal. However, changing them in 8.1 would mean anybody who enabled JIT using `jit_buffer_size` without also setting `jit` would implicitly disable it. This could be changed for master, but should be discussed on the ML as it is a breaking change.

Reply:
Is an RFC required, or just a simple email to discuss it OK?

Reply:
@danog Depends on whether there are any concerns. When there are concerns it's better to be safe and create a short RFC. Simple RFCs don't need to be long.

Reply:
@iluuu1994 Sent a discussion email + a karma request :)

Reply:
By the way, what about instead of completely removing the error, it simply indicated that JIT wasn't enabled because the buffer size is 0?

Reply:
I don't think the default configuration should emit a warning. It doesn't really, because `log_verbosity_level` needs to be increased. That's some INI weirdness by itself.